src/lavinmq/replication/server.cr

```diff
@@ -24,6 +24,7 @@ module LavinMQ
       @followers = Array(Follower).new
       @password : String
       @files = Hash(String, MFile?).new
+      @min_followers = 5_i64
```

Comment:
Very high default, I think 0 should be the default. And no way it has to be an `Int64`, I don't think many would need more than 2^31 followers. 
```suggestion
      @min_followers = 0
```